### PR TITLE
Dvpackager output

### PR DESCRIPTION
**Pull request by privatezero**
Ready for some testing I think on your end @dericed!

### CODE REVIEW

**Comment by dericed** [comment]
Hey @privatezero I added a few commits just to tweak the gui, still haven't tested this functions yet. Do you think that dvpackager should be a required dependency or optional one?

**Comment by privatezero** [comment]
These visual tweaks look great to me! Regarding dependencies - isn't it already [dependent on dvrescue](https://github.com/amiaopensource/homebrew-amiaos/blob/master/vrecord.rb#L17) (thus dvpackager)

**Comment by privatezero** [comment]
Just ran tests on a tape - functions seem  to work as expected for me

**Comment by privatezero** [comment]
anything still blocking this from merge?

**Comment by dericed** [comment]
minor conflict in the functions file

**Review by dericed — APPROVED** [review]
:)

File: vrecord [hunk 1]
```diff
@@ -883,6 +883,29 @@ AVFCTL_INPUT_GUI=$(cat << AVFCTL_FORM
...
+                <default>"${DV_RESCUE_OPTION_D}"</default>
+                <variable>DV_RESCUE_OPTION_D</variable>
+            </checkbox>
+            <checkbox>
+                <label>Split the output files on non-consecutive timecode values</label>
+                <default>"${DV_RESCUE_OPTION_T}"</default>
+                <variable>DV_RESCUE_OPTION_T</variable>
+            </checkbox>
+            <checkbox>
+                <label>Create technical subtitle track</label>
```
[the remark this answers is not in the document]
Comment (privatezero) [reply vrecord]: lol good point!

**Comment by privatezero** [comment]
@dericed I made some more changes to make the DV options in the GUI a little more conditional on install - work on your end?

**Comment by bturkus** [comment]
this is so awesome excited to try it out!

**Pull request merged**